### PR TITLE
Pass debugging flags from ratbagctl.devel to ratbagd

### PR DESCRIPTION
**Pull request by whot**

### CODE REVIEW

File: tools/ratbagctl.in [hunk 1]
```diff
@@ -1408,7 +1405,6 @@ class RatbagParserRoot(object):
             ns, rest = subparser.parse_known_args(rest, namespace=ns)
             if old_rest == rest:
                 break
-            pr_debug(ns, ns, rest)
```
Comment (bentiss) [inline tools/ratbagctl.in]: There are a few more pr_debug in RatbagParser.
You could also remove entirely the pr_debug function while you are at it.

Comment (whot) [reply tools/ratbagctl.in]: done and force-pushed

File: ratbagd/ratbagd.c [hunk 2]
```diff
@@ -520,10 +524,12 @@ int main(int argc, char *argv[])
 	int r;
 
 	if (argc > 1) {
-		if (streq(argv[1], "--verbose")) {
+		if (streq(argv[1], "--verbose=raw")) {
+			verbose = true;
```
Comment (Hjdskes) [inline ratbagd/ratbagd.c]: Shouldn't this be `verbose_raw = true`?

Comment (whot) [reply ratbagd/ratbagd.c]: doh. yes, thanks. pushed the fix directly to master

**Pull request merged**